deployment/kubernetes/aws/proxy.yaml

```diff
@@ -23,10 +23,11 @@ metadata:
   name: pulsar-proxy-config
 data:
   PULSAR_MEM: "\" -Xms4g -Xmx4g -XX:MaxDirectMemorySize=4g\""
-  PULSAR_PREFIX_brokerServiceURL: pulsar://broker:6650
-  PULSAR_PREFIX_brokerWebServiceURL: http://broker:8080
-  PULSAR_PREFIX_clusterName: pulsar-eks
-
+  brokerServiceURL: pulsar://broker:6650
+  brokerWebServiceURL: http://broker:8080
+  clusterName: us-east
+  zookeeperServers: zk-0.zookeeper,zk-1.zookeeper,zk-2.zookeeper
```

Comment:
I think you can just use the headless service name for `zookeeperServers` and `configurationStoreServers`. You don't need to write down all the pods here. because this will not work if people deploy a zookeeper with less than 3 pods.

Reply:
I just copied this from the bookkeeper.yaml, so I guess this should be modified in all of the other yaml's, right?

Reply:
yes correct

Reply:
I don't feel comfortable doing this change. All the other yaml examples including the helm charts list all the ZK servers and not just the headless service name.
I'm not familiar enough with Pulsar, ZK and k8s to be sure this is the right change.
Let me commit my change, which adheres to the existing "standard" and corrects the usability of the yaml (which didn't work before at all).

Reply:
sure. that can be done in a separate change.